tabs/osd.js

```diff
@@ -188,7 +188,7 @@ FONT.preview = function($el) {
   $el.empty()
   for (var i = 0; i < SYM.LOGO; i++) {
     var url = FONT.data.character_image_urls[i];
-    $el.append('<img src="'+url+'" title="0x'+i.toString(16)+'"/>');
+    $el.append('<img src="'+url+'" title="0x'+i.toString(16)+'"></img>');
```

Comment:
idk, pry fine to fix the following, but i just didnt want to mess w/ it. w/o this change i was getting:

![Uploading Screen Shot 2016-09-19 at 11.00.03 PM.png…]()